pkg/platform/kube/deployer.go

```diff
@@ -241,88 +240,17 @@ func (d *deployer) getFunctionPodLogs(namespace string, name string) string {
 		if getLogsErr != nil {
 			podLogsMessage += "Failed to read logs: " + getLogsErr.Error() + "\n"
 		} else {
-
 			scanner := bufio.NewScanner(logsRequest)
 
-			// get the last MaxLogLines logs
-			for scanner.Scan() {
-				currentLogLine, err := d.prettifyPodLogLine(scanner.Bytes())
-				if err != nil {
-
-					// when it is unstructured just add the log as a text
-					podLogsMessage += scanner.Text() + "\n"
-					continue
-				}
+			var formattedProcessorLogs string
+			formattedProcessorLogs, briefErrorMessage = d.platform.GetProcessorLogsAndBriefError(scanner)
```

Comment:
```suggestion
			formattedProcessorLogs, briefErrorMessage := d.platform.GetProcessorLogsAndBriefError(scanner)
```

and remove the var above?

Reply:
Unfortunately this is not possible - when performing `:=` every var on the left side must be newly declared - but in order for the `briefErrorMessage` to be recognized later we must declare it on the outer scope